### PR TITLE
CMDCT-159 UI tests (FaqAccordion and DashboardPage)

### PR DESCRIPTION
**Pull request by trancjessica**
### Description
This change adds an FaqAccordion test and increases the test coverage on the DashboardPage tests. 

### Related ticket(s)

CMDCT-159

https://jiraent.cms.gov/projects/CMDCT/issues/CMDCT-159?filter=allopenissues
---
### How to test
1. change directories to `services/ui-src`
2. run `yarn test` and `yarn test --coverage`
Note: coverage for the DashboardPage is not at 100 percent, but I wanted to put this code out since multiple people are working in the same test package

### Important updates

---
### Author checklist

- [✅] I have performed a self-review of my code
- [✅ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [N/A] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx [hunk 1]
```diff
@@ -87,19 +113,60 @@ describe("Test Report Dashboard view (Desktop)", () => {
...
+    });
+    mockMakeMediaQueryClasses.mockReturnValue("desktop");
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test("WP Dashboard renders table with empty text", async () => {
+    await act(async () => {
```
Comment (trancjessica) [inline services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx:146]: ope, we don't need async here, so i can remove this

File: services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx [hunk 2]
```diff
@@ -87,19 +113,56 @@ describe("Test Report Dashboard view (Desktop)", () => {
     await userEvent.click(callToActionButton);
     expect(screen.queryByText("Start new")).toBeVisible();
   });
+
+  test("Check that the SAR Dashboard view renders", async () => {
```
Comment (gmrabian) [inline services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx:117]: nit: don't need async here

File: services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx [hunk 3]
```diff
@@ -87,19 +113,56 @@ describe("Test Report Dashboard view (Desktop)", () => {
...
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test("WP Dashboard renders table with empty text", async () => {
+    render(wpDashboardWithNoReports);
+    expect(screen.getByText(wpVerbiage.body.empty)).toBeVisible();
+  });
+
+  test("SAR Dashboard renders table with empty text", async () => {
```
Comment (gmrabian) [inline services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx:150]: nit: don't need async on beforeEach or tests here

File: services/ui-src/src/utils/testing/mockFullReportJSON.tsx [hunk 4]
```diff
@@ -28,7 +28,7 @@ export const mockFullReportJSON: ReportJson = {
               props: {
                 href: "https://www.govinfo.gov/content/pkg/PLAW-109publ171/pdf/PLAW-109publ171.pdf",
                 target: "_blank",
-                ariaLabel: "Link opens in new tab",
+                "aria-label": "Link opens in new tab",
```
Comment (gmrabian) [inline services/ui-src/src/utils/testing/mockFullReportJSON.tsx:31]: suggestion: include visual content in label so it gets a full readout, or move to separate PR

File: services/ui-src/src/utils/testing/mockFullReportJSON.tsx [hunk 5]
```diff
@@ -926,7 +926,7 @@ export const mockFullReportJSON: ReportJson = {
...
```
Comment (gmrabian) [inline services/ui-src/src/utils/testing/mockFullReportJSON.tsx:929]: suggestion: include visual content in label so it gets a full readout, or move to separate PR

**Review by gmrabian — COMMENTED** [review]
Tests pass! I've included some suggestions and nits 💡

File: services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx (hunk 3 above)
Comment (trancjessica) [reply services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx:150]: 👀 tru, removing 🐈‍⬛

**Comment by trancjessica** [comment]
it looks like one of the checks is failing because my branch name has uppercase letters (and s3 buckets can't be named with uppercase letters). I am creating a new branch to fix this and will stand up a new PR :)

**Comment by trancjessica** [comment]
> it looks like one of the checks is failing because my branch name has uppercase letters (and s3 buckets can't be named with uppercase letters). I am creating a new branch to fix this and will stand up a new PR :)

https://github.com/Enterprise-CMCS/macpro-mdct-mfp/pull/153
